renewer/tasks/iam.py

```diff
@@ -67,3 +68,33 @@ def upload_certificate(session, operation_id: int, instance_type: RouteType):
 
     session.add(certificate)
     session.commit()
+
+
+@huey.retriable_task
+def delete_old_certificate(session, operation_id: int, instance_type: RouteType):
+    Operation: TOperation
+    Certificate: TCertificate
```

Comment:
yep. we have to be explicit here because mypy gets worried when it sees two different types that this might be later on. (`TOperation` and `TCertificate` are just unions of those two pairs)